### ISSUE TITLE
Unexpected error on render <CachedImage />

### ISSUE THREAD

**Issue by joaom182** [kfiroo/react-native-cached-image#129]
![screen shot 2018-07-19 at 03 14 04](https://user-images.githubusercontent.com/3612098/42924731-dfd19646-8b01-11e8-9625-d96ec7cf3b1f.png)

The problem ocur on fire multiple times component to load same image uri.

<CachedImage source={{ uri: 'IMAGE_URL' }} />

**Comment by joaom182** [comment]
This issue is caused by the latest version is using `react-native-fetch-blob` instead of using `rn-fetch-blob`.

@fungilation makes a [fork](https://github.com/fungilation/react-native-cached-image) that's correct the dependency, so i sugest to use this fork.

1 .`npm uninstall react-native-cached-image --save`
2. Check if the folder `react-native-cached-image` still on `node_modules` and remove completely.
3. `npm install react-native-fetch-blob --save`
4. `react-native link react-native-fetch-blob`
5. `npm install fungilation/react-native-cached-image --save`
6. ❤️

**Comment by JensDebergh** [comment]
If `react-native-fetch-blob` changed their name to `rn-fetch-blob` should we just merge @fungilation's fork and merge it into master?

The library `react-native-cached-image` is completely broken right now on latest versions of `react-native`.


`react-native-fetch-blob` is officially discontinued in favor of `rn-fetch-blob`

**Comment by ARMATAV** [comment]
@JensDebergh yes please - this is causing all sorts of issues, including breaking release builds for Android

**Comment by cyqui** [comment]
The project according to the contributors charts and the last commits seem rather dead,so unfortunately i have the feeling the PR is not going to be merged any soon. 

https://github.com/kfiroo/react-native-cached-image/graphs/contributors

@kfiroo 
Would be great if the README could mention to use fungilation/react-native-cached-image or any more popular fork you'll recommend

**Comment by venux92** [comment]
@joaom182 Great, been trying to debug it all day you just made my day

**Comment by andho** [comment]
netinfo package is needed now.
https://github.com/fungilation/react-native-cached-image/commit/363dea9b4e37a605faa15c5ea4dd502f98736831

**Comment by fungilation** [comment]
I know. See my last commit on my fork, tested to work: https://github.com/fungilation/react-native-cached-image/commits/master

**Comment by fungilation** [comment]
On topic of maintenance and forking: https://github.com/kfiroo/react-native-cached-image/issues/160
